app/src/assets/branding/credential-branding.ts

```diff
@@ -47,116 +57,101 @@ const digitalIdInvitationCardOverlay = {
   },
 }
 
-const digitalIdCardOverlay = {
-  imageSource: require('./service-bc-id-card.png'),
-  header: {
-    imageSource: require('./service-bc-header-logo.png'),
-    color: '#FFFFFF',
-    mapping: {
-      connectionLabel: 'Service BC',
-      credentialLabel: 'Person',
-    },
-  },
-  footer: { color: '#FFFFFF' },
-}
-
-const idCardBundle = {
-  capture_base: {},
-  overlays: [
-    {
+const createPersonCredentialBundle = (backgroundImageSource: any, verified = true) => {
+  const metaOverlays = []
+  if (verified) {
+    metaOverlays.push({
```

Comment:
Don't know if it's more effective or not but you can short hand this to `metaOverlays = [...metaOverlays, a, b]` perhaps a bit more declarative.

```
elcome to Node.js v16.14.0.
Type ".help" for more information.
> a = [1,2,3]
[ 1, 2, 3 ]
> [...a, 4, 5]
[ 1, 2, 3, 4, 5 ]
> 
```

Reply:
Yes. I am aware of the spread operator. I use that syntax for making copies. I would also need to change from `const ` to `let`. I decided to keep it that way for now. This will eventually go away, and get published somewhere else.

Reply:
But that did got me to take a look and notice that I actually forgot to actually use the array as part of the return statement.